package.json

```diff
@@ -37,11 +37,12 @@
     "extra-types",
     "shared",
     "webview-ui",
+    "agentic",
```

Comment:
Be aware that order in the workspaces list here matters for any script that has `npm ... --workspaces` calls .  If agentic is referenced by webview-ui at all, it should come before it.